wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSettings.php

```diff
@@ -40,7 +40,6 @@ public function collectionSettingsCreateAdminPage()
 
         <div class="wrap">
             <h1><?php _e('Site Settings', 'cds-snc') ?></h1>
-            <?php settings_errors(); ?>
```

Comment:
Wondering why this was removed.

Reply:
Is this why the double messages were showing?

Reply:
Yeah, removed this and the double errors/confirmation messages went away.